website/docs/developer/research/research-orca-delete-functionality.md

```diff
@@ -0,0 +1,122 @@
+---
+id: research-orca-delete-functionality
+title: Research Notes on ORCA delete functionality.
+description: Research Notes on implementing delete functionality for ORCA.
+---
+
+## Delete functionality use cases
+
+Some use cases for delete functionality are as follows:
+
+- As a DAAC, I would like remove data from ORCA that is invalid or expired in order to maintain a clean data archive.
+
+- As a DAAC, I would like to remove data from ORCA only if it has been in glacier for more than 90 days from initial ingest to keep from incurring additional cost for early removal.
+
+    This can be implemented by using soft delete. We can either flag the data in the database with a new column named `Pending Deletion` or we can create a separate table that contains those data marked for deletion. Then after 90 days, we can permanently delete from the DB as well as the glacier bucket.
+
+
+- As a DAAC, I would like to configure the amount of time data remains in ORCA after being signaled to be deleted.	
+
+    This should be configurable by each DAAC that is using ORCA. Even after the data is deleted from Cumulus, it should stay in ORCA for x number of days for safety reasons.
+
+
+- As a DAAC, I would like only authorized personnel to be able to remove data from ORCA in a controlled way to minimize risk of accidental or malicious deletions.
+
+    This could be implemented with the use of NASA launchpad authentication. This [link](https://idmax.nasa.gov/nams/asset/255115) is used to submit a new GSFC workflow request and most probably will be used to only allow specific users to perform the deletes. Contact `robert.a.meredith@nasa.gov` and `tadd.buffington@nasa.gov` with any issues regarding launchpad integration.
+
+- As a DAAC, I would like duplicate data to be versioned in ORCA in order to recover the proper version of the data based on the type of failure.	
+
+
+- Perform a check with CMR and/or Cumulus to verify that the granule has been deleted from those locations prior to allowing the delete to proceed in ORCA.
+
+    
+    This can be implemented in the existing lambda function that will be used to delete the data from ORCA.
+
```

Comment:
A drawing would be good here to help show the architecture you are thinking of.

Reply:
yes I was thinking of that. Should I use draw.io and do I need certain access on that?

Reply:
That will be fne, Ideally you would use Lucidchart but if you do not have access to that app yet, DrawIO will work fine to create and save an image.